### PR TITLE
Add section on `reset -p` to unstage edits

### PR DESCRIPTION
**Pull request by nielsonm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -381,6 +382,11 @@ Then, you will need to use the `e` option to manually choose which lines to add.
 
 `git add` will add the entire file to a commit. `git add -p` will allow to interactively select which changes you want to add.
 
+<a href="selective-unstage-edits"></a>
+### I staged too many edits, and I want to break them out into a separate commit
+
+`git reset -p` will open a patch mode reset dialog.  Much like `git add -p` but yes will reset the change, unstaging it.
```
Comment (RichardLitt) [inline README.md]: But yes will?

Comment (RichardLitt) [reply README.md]: This is hard for me to parse. Please make the text clearer. Thanks.

Comment (nielsonm) [reply README.md]: Understood. Took a minute to remember exactly what I was thinking when I wrote this.

**Comment by RichardLitt** [comment]
Thank you, @nielsonm! I rebased, edited slightly, squashed, and merged in 8b09e3f. :)

**Pull request closed without merging**